repos/corefx.common.props

```diff
@@ -41,6 +41,8 @@
     <BuildArguments>$(BuildArguments) $(FlagParameterPrefix)binaryLog</BuildArguments>
     <BuildArguments>$(BuildArguments) $(FlagParameterPrefix)ci</BuildArguments>
     <BuildArguments Condition="'$(OS)' != 'Windows_NT'">$(BuildArguments) -warnAsError:0</BuildArguments>
+    <!-- some projects still set TreatWarningsAsErrors, override them -->
+    <BuildArguments Condition="'$(OS)' != 'Windows_NT'">$(BuildArguments) /p:TreatWarningsAsErrors=false</BuildArguments>
```

Comment:
Should we track why this became necessary?

Reply:
:+1: I generally file issues/PRs to fix these sort of build failures.

Reply:
Logged https://github.com/dotnet/source-build/issues/1601 for this.